### PR TITLE
Update dev setup

### PR DESCRIPTION
**Pull request by hblitza**
- adds print servlet including  4 template
- adds start-config for masterportal
- update readme

### CODE REVIEW

File: common-services.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: '3.9'
+services:
+  nginx:
+    container_name: ${CONTAINER_NAME_PREFIX}-nginx
+    image: nginx:1.25.3-alpine
+    volumes:
+      - ./nginx/ssl/private/localhost.crt:/etc/nginx/ssl/private/localhost.crt
```
Comment (ahennr) [inline common-services.yml:7]: Is this a self-signed certificate? If so, how are these created?
If not required, I would prefer to remove this

File: setup.sh [hunk 2]
```diff
@@ -6,7 +6,9 @@ set -e
 USER_ID=$(id -u)
 GROUP_ID=$(id -g)
 USER_NAME=$(whoami)
-MASTERPORTAL_VERSION=v2.33.0
+MASTERPORTAL_VERSION=v3.0.0-beta2
```
Comment (ahennr) [inline setup.sh:9]: Is the beta version wanted here?

File: nginx/dev/default.conf [hunk 3]
```diff
@@ -38,5 +49,5 @@ server {
     add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
     add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
     add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
-  }
+  }
```
Comment (ahennr) [inline nginx/dev/default.conf]:
```suggestion
  }
```

**Review by ahennr — APPROVED** [review]

File: common-services.yml (hunk 1 above)
Comment (hblitza) [reply common-services.yml:7]: yes self signed. via `setup.sh` like in shogun-docker repo

**Pull request merged**